wiki/content/tutorial-7/index.md

```diff
@@ -292,20 +292,11 @@ Along with the existing features to query and search string predicates, the addi
 This marks the end of our three tutorial-streak exploring string indices and their queries
```

Comment:
streak? sorry, I just had some hilarious sporting connotations come to mind...in any case, please remove the dash "-"